app/build.gradle

```diff
@@ -53,16 +52,17 @@ android {
 }
 
 ext {
-    supportLibVersion = "26.0.1"
-    espressoVersion = "2.2.2"
+    supportLibVersion = "26.1.0"
+    espressoVersion = "3.0.1"
     androidTestVersion = "0.5"
     hamcrestVersion = "1.3"
     junitVersion = "4.12"
 }
 
 dependencies {
-    kapt "com.android.databinding:compiler:2.3.2"
+    kapt "com.android.databinding:compiler:3.0.1"
 
+    //noinspection GradleDependency
```

Comment:
this is useful warning since it highlights when versions are outdated - how comes you disabled it?

Reply:
True. That has to be there.